Production/python/Production.py

```diff
@@ -192,7 +202,9 @@
 
     lheEventProduct    = cms.InputTag('externalLHEProducer'),
     genEvent           = cms.InputTag('generator'),
-    genParticles       = cms.InputTag('prunedGenParticles'),
+    genParticles       = cms.InputTag('genParticlePlusGeant'),
+    genPlusSimParticleMode   = cms.bool(True),
```

Comment:
this should be optional and set with external parameter whatever to use standard particle collection or +geant